apps/07jquery-tic-tac-toe/index.html

```diff
@@ -22,7 +22,8 @@
     <div data-cell="8"></div>
   </div>
   <div id="announce-winner"></div>
-  <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
+  <button id="clear">Clear Board</button>
+  <script type="text/javascript" src="../vendor/jquery.js"></script>
```

Comment:
Sorry that was assuming we want the same version of jquery for the app and the jsdom tests. We could just use npm for the jsdom version and a cdn for the app version. Jsdom would have to whitelist the cdn version though